### DISCUSSION TITLE
can not recognize \makecvheader

### DISCUSSION THREAD

**Discussion by anhnt2407** [posquit0/Awesome-CV#520] [Q&A]
i downloaded your original source code and tried to compile in texStudio but failed. What's wrong here bro?

Undefined control sequence. \makecvheader[C]
Undefined control sequence. \makecvheader[C]
Undefined control sequence. \makecvheader[C]
Overfull \hbox (1.45999pt too wide) in paragraph
Underfull \hbox (badness 10000) in paragraph
Underfull \hbox (badness 10000) in paragraph
Underfull \hbox (badness 10000) in paragraph
Underfull \hbox (badness 10000) in paragraph
Underfull \hbox (badness 10000) in paragraph
Underfull \hbox (badness 10000) in paragraph
Underfull \hbox (badness 10000) in paragraph

**Comment by justinbass** [comment]
Using xelatex?

**Comment by oozypal** [comment]
Yes. I have the same problem. It happens when using xelatex.

**Comment by oii-nasif** [comment]
@oozypal Kindly recheck the links on header section. It might happen when the links are dead.

**Comment by droundy** [comment]
@oii-nasif Can you clarify what you mean? Which links?
